code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -338,16 +338,20 @@ CONTENTS:
 	add_attack_logs(user, L, "Stunned with [src]")
 
 /obj/item/abductor_baton/proc/SleepAttack(mob/living/L,mob/living/user)
-	if(L.IsStunned() || L.IsSleeping())
-		L.visible_message("<span class='danger'>[user] has induced sleep in [L] with [src]!</span>", \
-							"<span class='userdanger'>You suddenly feel very drowsy!</span>")
-		playsound(loc, 'sound/weapons/egloves.ogg', 50, TRUE, -1)
-		L.Sleeping(120 SECONDS)
-		add_attack_logs(user, L, "Put to sleep with [src]")
-	else
-		L.AdjustDrowsy(2 SECONDS)
-		to_chat(user, "<span class='warning'>Sleep inducement works fully only on stunned specimens!</span>")
-		L.visible_message("<span class='danger'>[user] tried to induce sleep in [L] with [src]!</span>", \
+	if(!iscarbon(L))
+		return
+	var/mob/living/carbon/C = L
+	if(do_mob(user, C, 25))
```

Comment:
Also make this into an early return please
```suggestion
	if(do_mob(user, C, 2.5 SECONDS))
```

Reply:
ill bring the cuff code up to this standard too on next commit